### PR TITLE
Some fixes to improve code quality

### PR DESCRIPTION
**Pull request by Huluti**
- Fix unused parameter in string format and missing used of parameter in string format
- Use more pythonic way to iterate
- Use type() instead of isinstance() for a typecheck

### CODE REVIEW

File: dbr/wizard.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def SetPages(self, pages):
         
         self.ClearPages() # Remove any current pages from the wizard
         
-        if type(pages) not in (list, tuple):
+        if isinstance(pages, (list, tuple)):
```
Comment (AntumDeluge) [inline dbr/wizard.py]: Shouldn't this be "if not isinstance..."?

File: dbr/log.py [hunk 2]
```diff
@@ -153,7 +153,7 @@ def SetLogLevel(self, l_level):
             self.log_level = l_level
             log_set = True
         
-        elif type(l_level) in (unicode, str):
+        elif isinstance(l_level, (unicode, str))
```
Comment (AntumDeluge) [inline dbr/log.py]: Missing colon (:) here.

File: dbr/about.py [hunk 3]
```diff
@@ -312,7 +312,7 @@ def AddJobs(self, name, jobs, email=wx.EmptyString):
             Logger.Debug(__name__, GT(u'Converting string argument "jobs" to tuple'))
             jobs = (jobs,)
         
-        for x, value in enumerate(jobs):
+        for value in jobs:
             next_item = self.credits.GetItemCount()
             if x == 0:
```
Comment (AntumDeluge) [inline dbr/about.py]: 'x' not defined.

**Comment by AntumDeluge** [comment]
I'm sorry it took me a couple days before I was able to review this.

File: dbr/wizard.py (hunk 1 above)
Comment (Huluti) [inline dbr/wizard.py]: You are right...

File: dbr/log.py (hunk 2 above)
Comment (Huluti) [inline dbr/log.py]: Good job !

**Pull request merged**